charts/trusted-attestation-controller/templates/deployment.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: tac-app
+  namespace: {{ .Values.nameSpace }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      labels:
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --leader-elect
+        - --registration-path=/registration/controller.sock
+        command:
+        - /manager
+        image: intel/trusted-attestation-controller:{{ .Values.deployment.imageTag }}
```

Comment:
don't hardcode anything on the image. Use similar structure as in TCS issuer

```yaml
image:
  hub:
  name:
  tag:
```
In values.yaml leave the tag "", and default to Chart.appVersion.